### PR TITLE
Expose p2p port in docker-compose setup

### PR DESCRIPTION
**Pull request by dapplion**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@ services:
     restart: always
     volumes:
       - lodestar:/root/.local/share/lodestar
-    command: beacon --network ${NETWORK:-pyrmont}
+    command: beacon --network ${NETWORK:-mainnet}
```
Comment (3xtr4t3rr3str14l) [inline docker-compose.yml]: should this maybe be changed back to pyrmont?

File: docker-compose.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ services:
     volumes:
       - lodestar:/root/.local/share/lodestar
     command: beacon --network ${NETWORK:-pyrmont}
+    ports:
+      - "9000:9000"
```
Comment (twoeths) [inline docker-compose.yml:10]: this is disc5 port, and I still see we can detect new peers without this so I'm curious how could this help?

Comment (dapplion) [reply docker-compose.yml:10]: If you don't expose this port you will get only outbound peers. Opening the p2p port so other nodes can connect to you is very recommended and necessary to test a network participant with inbound and outbound connections

Comment (dapplion) [reply docker-compose.yml:10]: https://docs.prylabs.network/docs/prysm-usage/p2p-host-ip/#port-forwarding
https://lighthouse-book.sigmaprime.io/docker.html?highlight=port#ports

**Review by twoeths — APPROVED** [review]

**Pull request merged**